.github/workflows/ci.yml

```diff
@@ -47,6 +47,20 @@ jobs:
         uses: FreeRTOS/CI-CD-Github-Actions/complexity@main
         with:
           path: ./
+  doxygen:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Download MQTT tag
+        run: |
+          # We don't need to generate the coreMQTT docs, we only need the tag file. We can just download it.
```

Comment:
There can be issue in the future when changes to coreMQTT do not match the documentation present in a previous release. Would it be better to just build the tag off of the coreMQTT commit submoduled in the repository instead?

Reply:
I disagree. It would make sense for the tag file to match the documentation it was generated with; if there's a coreMQTT update, I don't think building the documentation from the coreMQTT commit would help since it's still linking to the external docs.

Reply:
Wouldn't changes to coreMQTT still work as long as they are re-hosted in the subsequent release?

Reply:
Yes, but if the docs are re-hosted, then the tag file from the URL will also be up to date since the tag file is part of the generated docs.